osi_sensorviewconfiguration.proto

```diff
@@ -729,6 +729,52 @@ message CameraSensorViewConfiguration
     //
     repeated WavelengthData wavelength_data = 11;
 
+    // Indicates if and how the the pixel data is ordered
+    //
+    // By default, the pixel data is is left-to-right, top-to-bottom. 
+    // 
+    // \note for rotations of the pixel data, use the camera coordinate system
+    //
+    optional PixelOrder pixel_order = 12;
+
+    // Pixel layout 
+    //
+    // Pixel layout documents the order of pixels in the \c image_data
+    // in CameraSensorView. 
+    //
+    // \note this enum does not contain an entry to do mirroring upside down
+    // and left-to-right at the same time, because this is equivalent to a 180°
+    // rotation, which should be indicated in the sensor coordinate system 
+    //
+    enum PixelOrder
+    {
+        // Pixel order is unknown (must not be used).
+        //
+        PIXEL_ORDER_UNKNOWN = 0;
+
+        // Unspecified but unknown pixel order.
```

Comment:
Usually if "other" is used, it is referred to a known (not unknown) thing, e.g. a known pixel order that is not specified in OSI.

Reply:
ok, i have fixed this in the follow-up commit [a1fc975](https://github.com/OpenSimulationInterface/open-simulation-interface/pull/698/commits/a1fc9757224f356a13f3dba377a9857c0af76e37)